cac-prod.yml

```
@@ -55,7 +55,7 @@ jenkins:
         title: SSCS Dashboard
     - buildMonitor:
         includeRegex: >-
-          ^HMCTS_.*IAC.*\/(ia-case-api|ia-shared-infrastructure|ia-ccd-e2e-tests)\/master
+          ^HMCTS_.*IAC.*\/(ia-case-api|ia-case-documents-api|ia-case-notifications-api|ia-shared-infrastructure|ia-ccd-e2e-tests)\/master
```

@Rob-Doherty any reason why you don't add all of them like most teams?
```suggestion
          ^HMCTS_.*IAC.*\/.+\/master
```

---

I was going to, but there are additional repos that aren't used anymore, e.g. `ia-appeal-frontend` that I wanted to avoid. However I've just checked with @fantasticjamieburns and it sounds like it shouldn't be a problem to make it generic, so thanks for the suggestion @timja 👍